build-scripts/build-auth-rpm

```diff
@@ -340,6 +340,15 @@ Requires: %{name}%{?_isa} = %{version}-%{release}
 %description backend-lua
 This package contains the lua backend for %{name}
 
+%package backend-lua2
+Summary: New Lua backend for %{name}
+Group: System Environment/Daemons
+Requires: %{name}%{?_isa} = %{version}-%{release}
+%global backends %{backends} lua2
+
+%description backend-lua2
+This package contains the new lua backend for %{name}
+
```

Comment:
I'm pretty sure you also need to tell the RPM what files go into the package.

Reply:
I hope I finally found all them places. =)